packages/_site/src/pages/components/heading/index.tsx

```diff
@@ -0,0 +1,383 @@
+import ComponentHero from "../commons/component-hero";
+import stylex from "@stylexjs/stylex";
+import ExampleBlock from "../commons/example-block";
+import CodeBlock from "../commons/code-block";
+import { PreviewBlock } from "../commons/preview_block";
+import ComponentInfo from "../commons/component-info";
+import { Divider } from "@controlkit/divider";
+import { H1, H2, H3, H4, H5, H6 } from "@controlkit/headings";
+import { Tabs, TabsList, TabsTrigger, TabsContent } from "@controlkit/tabs";
+
+const headingPageStyles = stylex.create({
+	wrapper: {
+		width: "100%",
+		height: "100%",
+		padding: "2rem",
+		boxSizing: "border-box",
+	},
+
+	threeColumnLayout: {
+		display: "flex",
+	},
+
+	middleColumn: {
+		width: "100%",
+		height: "100%",
+		//padding: "1rem",
+		marginTop: "2rem",
+		boxSizing: "border-box",
+		display: "flex",
+		flexDirection: "column",
+		gap: "2rem",
+	},
+
+	codeWrapper: {
+		// border: "1px solid var(--border-100)",
+		borderRadius: "var(--border-radius)",
+		backgroundColor: "#121212",
+		// padding: "1rem",
+		boxSizing: "border-box",
+	},
+
+	blockWrapper: {
+		marginTop: "1rem",
+	},
+
+	stepBlock: {
+		display: "flex",
+		flexDirection: "column",
+		gap: "1rem",
+	},
+});
+
+export default function HeadingPage() {
+	return (
+		<div {...stylex.props(headingPageStyles.wrapper)}>
+			<ComponentHero
+				title="Heading"
+				description="A heading is used as a title/subtitle element for pages."
+			/>
+
+			<div {...stylex.props(headingPageStyles.threeColumnLayout)}>
+				{/* <div style={{ backgroundColor: "red", width: "fit-content" }}>
+          scroller
+        </div> */}
+
+				<div {...stylex.props(headingPageStyles.middleColumn)}>
+					<H2>Installation</H2>
+					<Divider />
+					<div>
+						<Tabs defaultValue="tab-1">
+							<TabsList>
+								<TabsTrigger value="tab-1">Automatic</TabsTrigger>
+								<TabsTrigger value="tab-2">Manual</TabsTrigger>
+							</TabsList>
+							<TabsContent value="tab-1">
+								<div {...stylex.props(headingPageStyles.blockWrapper)}>
+									<ComponentInfo
+										install="pnpm add @controlkit/heading"
+										npmjs="https://www.npmjs.com/package/@controlkit/heading"
+										npmTitle="@controlkit/heading"
+										source="https://github.com/clearfeld/control-ui/tree/main/packages/heading"
```

Comment:
The package is called `@controlkit/headings` note the `s` at the end. Fix that and LGTM for merge.